CHANGELOG.md

```diff
@@ -6,16 +6,18 @@
 ### Fixed
 - Github template tests fail in proxy environment ([#56](https://github.com/opendevstack/ods-document-generation-svc/issues/56))
 
-## [3.0] - 2020-08-11
```

Comment:
Have a look at the changes in the changelog of the ods-jenkins-shared library if we release this with the 4.x it should have a section

Reply:
but I need to fix the CHANGELOG in this PR or the next PR against the 4.x branch?